src/use-dropdown-menu.ts

```diff
@@ -86,6 +86,25 @@ export default function useDropdownMenu(itemCount: number) {
 		return () => document.removeEventListener('click', handleEveryClick);
 	}, [isOpen]);
 
+	// Disable scroll when the menu is opened, and revert back to the previous style when the menu is closed
+	useEffect(() => {
+		const disableArrowScroll = (event: KeyboardEvent) => {
+			if (
+				isOpen &&
+				(event.key === 'ArrowDown' ||
+					event.key === 'ArrowUp' ||
+					event.key === 'ArrowLeft' ||
+					event.key === 'ArrowRight')
```

Comment:
Do we actually have any listeners for the left/right arrow keys? If not, I think we should only disable scrolling for the up/down arrow keys.

Reply:
We don't, it could be out of scope for this hook to be preventing those arrow keys, I'll take it out.